src/app/components/DownloadKMP.svelte

```diff
@@ -3,25 +3,32 @@
 </script>
 
 <style>
-  .download-link {
+  .download-component--disabled {
+    cursor: not-allowed;
+  }
+  .download-component__link {
     display: block;
     margin-top: 1.5em;
   }
 
-  .download-link--disabled {
+  .download-component__link--disabled {
     color: var(--gray-medium-dark);
-    cursor: not-allowed;
     opacity: 25%;
+    pointer-events: none;
     text-decoration: none;
   }
 </style>
 
-<a
-  href={downloadURL ? downloadURL : '#'}
-  download="Example.kmp"
-  class="download-link"
-  class:download-link--disabled={downloadURL == ''}
-  data-cy="download-kmp"
-  data-download-state={downloadURL ? 'ready' : 'disabled'}>
-  Download KMP Package
-</a>
+<div
+  class="download-component"
+  class:download-component--disabled={downloadURL == ''}>
+  <a
+    href={downloadURL ? downloadURL : '#'}
+    download="Example.kmp"
+    class="download-component__link"
+    class:download-component__link--disabled={downloadURL == ''}
+    data-cy="download-kmp"
+    data-download-state={downloadURL ? 'ready' : 'disabled'}>
+    Download KMP Package
+  </a>
+</div>
```

Comment:
Is the wrapper `<div>` necessary? Why?

Reply:
Yup, the wrapper `<div>` is to inherit the style of `cursor: not-allowed;` while the `<a>` will only responsible for `pointer-events: none`. Since if `cursor: not-allowed;` & `pointer-events: none` exist together as a style for a single tag, only one of them will be selected, depending on the order. This was the reason why before the fix it will only show disabled cursor but the user is still able to click on the `<a>` tag.  

The behaviour now will be the wrapper class `<div>` giving the style of `cursor: not-allowed;` and the `<a>` tag being refrained from clicking by `pointer-events: none;`

Reply:
Oh wow! Yeah, if the `<a>` gets `pointer-events: none` it can't get... pointer events... including the event that changes the cursor!